src/lib/blocks.js

```diff
@@ -100,6 +100,14 @@ export default function (vm) {
         this.jsonInit(json);
     };
 
+    ScratchBlocks.Blocks.motion_glideto_menu.init = function () {
```

Comment:
Ah, I see. Well, I don’t know how to make Travis realize that, so the scratch-blocks PR needs to be merged first then.